### ISSUE TITLE
A filter crash. Not the same as the other crash. (Sorry.) :P

### ISSUE THREAD

**Issue by NJM1564** [Direwolf20-MC/JustDireThings#124]
Data filter crashed to menu.
No crash report but here is a log file.
[latest.log](https://github.com/user-attachments/files/16543307/latest.log)
And the error message. (I was not going to wright all that gobbelty gook down. :p)
![Screenshot (10820)](https://github.com/user-attachments/assets/7267fca7-0cf8-466b-8485-30e94c8c2b7f)

**Comment by Direwolf20-MC** [comment]
This looks like an error from laserio? Can you let me know what you were doing?

**Comment by NJM1564** [comment]
Sure. I open pack right click data filter and it crashes.
I's running ATM 10 version 0.28
With the fallowing changes.
Updated: 
ApothicEnchanting-1.21-1.1.2
Added mods: 
RingOfRepair-NeoForge-1.21-1.8.4
RingOfGrowth-NeoForge-1.21-1.8.2
travelersbackpack-neoforge-1.21-10.0.5
OreBerries-Replanted-1.21-0.9.4
toms_storage-1.21-2.0.8
GrowableOres-2.3.0-1.21-NeoForge
squatgrow-neoforge-21.0.3+mc1.21.disabled
ironchest-1.21-neoforge-16.0.7

I also noticed this pop up when logging back in and looking at NEI.
![Screenshot (10826)](https://github.com/user-attachments/assets/74d7b481-7e47-4ba4-be1d-e35c4080a5a8)
